### PR TITLE
Use PostgreSQL identity columns instead of serial

### LINKED ISSUE #74: Use identitiy columns instead of serial for Primary Keys

### PR DESCRIPTION
**Pull request by kevincianfarini**
Closes #74

### CODE REVIEW

File: database/ccm_schema.sql [hunk 1]
```diff
@@ -616,19 +616,14 @@ CREATE TABLE public.action_contact_legislator (
 -- Name: action_contact_legislator_id_seq; Type: SEQUENCE; Schema: public; Owner: -
 --
 
-CREATE SEQUENCE public.action_contact_legislator_id_seq
+ALTER TABLE public.action_contact_legislator ALTER COLUMN id ADD GENERATED ALWAYS AS IDENTITY (
```
Comment (EvanHahn) [inline database/ccm_schema.sql:619]: This file looks autogenerated so I didn't review it. Lemme know if that's wrong.

File: backend/src/test/kotlin/org/climatechangemakers/act/feature/action/manager/DatabaseActionTrackerManagerTest.kt [hunk 2]
```diff
@@ -14,7 +14,7 @@ class DatabaseActionTrackerManagerTest : TestContainerProvider() {
   private val manager = DatabaseActionTrackerManager(database, EmptyCoroutineContext)
 
   @Test fun `email action entries are recorded in both tables`() = suspendTest {
-    driver.insertIssue(1, "this is an issue", "tweet", "url.com")
+    val id = driver.insertIssue("this is an issue", "tweet", "url.com")
     driver.insertMemberOfCongress(DEFAULT_MEMBER_OF_CONGRESS.copy(bioguideId = "hello"))
     manager.trackActionSendEmail("foo@foo.com", "hello", 1)
```
Comment (EvanHahn) [inline backend/src/test/kotlin/org/climatechangemakers/act/feature/action/manager/DatabaseActionTrackerManagerTest.kt:19]: Do we need to update the last argument here?

```suggestion
    manager.trackActionSendEmail("foo@foo.com", "hello", id)
```

Same comment applies to some of the calls below.

File: backend/src/test/kotlin/org/climatechangemakers/act/feature/action/manager/DatabaseActionTrackerManagerTest.kt [hunk 3]
```diff
@@ -36,12 +36,12 @@ class DatabaseActionTrackerManagerTest : TestContainerProvider() {
...
 
   @Test fun `recording a legislator call insert into both tables`() = suspendTest {
-    driver.insertIssue(1, "issue", "tweet", "url.com")
+    val id = driver.insertIssue("issue", "tweet", "url.com")
     driver.insertMemberOfCongress(DEFAULT_MEMBER_OF_CONGRESS.copy(bioguideId = "bioguide"))
     manager.trackActionPhoneCall("foo@foo.com", "bioguide", 1, "8675309")
 
     assertEquals(
-      1,
+      id,
```
Comment (EvanHahn) [inline backend/src/test/kotlin/org/climatechangemakers/act/feature/action/manager/DatabaseActionTrackerManagerTest.kt:44]: It seems weird that we'd check that the `COUNT(*)` is the same as the ID...should this be reverted?

**Review by EvanHahn — APPROVED** [review]
LGTM. Found a few things I didn't understand but I think things are fine.

File: database/ccm_schema.sql (hunk 1 above)
Comment (kevincianfarini) [reply database/ccm_schema.sql:619]: This is indeed autogenerated!

File: backend/src/test/kotlin/org/climatechangemakers/act/feature/action/manager/DatabaseActionTrackerManagerTest.kt (hunk 2 above)
Comment (kevincianfarini) [reply backend/src/test/kotlin/org/climatechangemakers/act/feature/action/manager/DatabaseActionTrackerManagerTest.kt:19]: yes, good call.

File: backend/src/test/kotlin/org/climatechangemakers/act/feature/action/manager/DatabaseActionTrackerManagerTest.kt (hunk 3 above)
Comment (kevincianfarini) [reply backend/src/test/kotlin/org/climatechangemakers/act/feature/action/manager/DatabaseActionTrackerManagerTest.kt:44]: good catch!

**Pull request merged**